lib/services/account-api-service.js

```diff
@@ -0,0 +1,193 @@
+// Copyright 2016, EMC, Inc.
+
+'use strict';
+
+var di = require('di'),
+    nodeAcl = require( 'acl' );
+
+module.exports = accountApiServiceFactory;
+di.annotate(accountApiServiceFactory, new di.Provide('Http.Services.Api.Account'));
+di.annotate(accountApiServiceFactory,
+    new di.Inject(
+        'Services.Waterline',
+        'Errors',
+        'Logger',
+        '_',
+        'Promise',
+        'Constants',
+        'Services.Configuration',
+        'Assert'
+    )
+);
+function accountApiServiceFactory(
+    waterline,
+    Errors,
+    Logger,
+    _,
+    Promise,
+    Constants,
+    configuration,
+    assert
+) {
+    var logger = Logger.initialize(accountApiServiceFactory);
+
+    function AccountApiService() {
+        this.acl = new nodeAcl(new nodeAcl.memoryBackend());  // jshint ignore:line
+    }
+
+    AccountApiService.prototype.start = function() {
+        var acl = this.acl;
+        var self = this;
+        var endpoints = configuration.get('httpEndpoints', []);
+        var authEnabled = _(endpoints).filter(function(endpoint) {
```

Comment:
`every` wouldn't work, but `some` would.  `every` would only initialize the service if _all_ endpoints have auth enabled.